lib/inspec/metadata.rb

```diff
@@ -109,6 +110,12 @@ def valid
         next unless params[field.to_sym].nil?
         errors.push("Missing profile #{field} in #{ref}")
       end
+
+      # if version is set, ensure it is correct
+      if !params[:version].nil? && !valid_version?(params[:version])
+        errors.push("Version need to be in SemVer format in #{ref}")
+      end
+
       %w{ title summary maintainer copyright }.each do |field|
         next unless params[field.to_sym].nil?
         warnings.push("Missing profile #{field} in #{ref}")
```

Comment:
Since the `inspec check` output will provide the filename that generated the warning:

```
✖  /Users/aleff/profiles/test1/inspec.yml:0:0: Version need to be in SemVer format in inspec.yml
```

... I propose can just change this to "Version needs to be in SemVer format" to avoid the duplicate "inspec.yml" output